src/types/roles.ts

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+export type ChannelModerationRoles = 'members' | 'guests';
```

Comment:
Can this use the constants?

Reply:
I tried but it doesn't seem to work. I also couldn't find anywhere where we use constants for types. 
![image](https://user-images.githubusercontent.com/17804942/75598103-71775a00-5a67-11ea-90ab-25a07cdd2786.png)